remoteappmanager/handlers/home_handler.py

```diff
@@ -112,7 +112,9 @@ def _actionhandler_start(self, options):
 
         # The server is up and running. Now contact the proxy and add
         # the container url to it.
-        urlpath = yield self.application.reverse_proxy.add_container(container)
+        urlpath = self.application.urlpath_for_object(container)
```

Comment:
Why not `url_path_join(self.application.command_line_config.base_urlpath, container.urlpath)` here?
Note that `base_urlpath` is also retrievable in `HomeHandler.settings['base_urlpath']`